app/src/main/java/org/oppia/app/player/state/itemviewmodel/SelectionInteractionViewModel.kt

```diff
@@ -13,7 +13,7 @@ import org.oppia.app.viewmodel.ObservableArrayList
 
 /** [StateItemViewModel] for multiple or item-selection input choice list. */
 class SelectionInteractionViewModel(
-  val explorationId: String,
+  val entityId: String,
   interaction: Interaction,
   private val interactionAnswerReceiver: InteractionAnswerReceiver,
   interactionAnswerErrorReceiver: InteractionAnswerErrorReceiver
```

Comment:
`interactionAnswerErrorReceiver ` is not been used anywhere.

Reply:
Nice catch, removed.